src/modules/ekf2/EKF/covariance.cpp

```diff
@@ -200,15 +200,15 @@ void Ekf::predictCovariance(const imuSample &imu_delayed)
 
 #if defined(CONFIG_EKF2_WIND)
 
-	if (_control_status.flags.wind) {
-		// wind vel: add process noise
-		float wind_vel_nsd_scaled = math::constrain(_params.wind_vel_nsd, 0.f,
-					    1.f) * (1.f + _params.wind_vel_nsd_scaler * fabsf(_height_rate_lpf));
+	// wind vel: add process noise
+	if (!_external_wind_init) {
```

Comment:
I see that `_external_wind_init` is automatically set to false when starting airspeed fusion (https://github.com/PX4/PX4-Autopilot/pull/23277/files#diff-7da169004af9436e30e9e918ddd7c7621b30de929ea72261ebaed2205fca8f3fR139) but what about drag and sideslip?